src/main/features/certificateOfService/views/document-upload.njk

```diff
@@ -0,0 +1,124 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('Upload your documents') %}
+
+{% from "form.njk" import csrfProtection, saveAndContinueButton, errorSummary, submitButton %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      {% if not fileToUpload %}
+        <form novalidate method="post">
+          {{ csrfProtection(csrf) }}
+      {% endif %}
+        <p>{{ t('Use this page to send the County Court the following documents you’ve served on the defendant.') }}</p>
+
+        <div class="form-group">
+          <label class="form-label-bold">{{ t('Claim form') }}</label>
+          <label class="form-hint">{{ t('Not required') }}</label>
+        </div>
+
+        <div class="form-group">
+          <label class="form-label-bold">{{ t('Particulars of claim') }}</label>
+          {% if particularsOfClaim.length > 0 %}
+            <div class="form-group">
+              <ul class="list-number list-border">
+                {% for file in particularsOfClaim %}
+                  <li>
+                    <a target="_blank" href="/legal/certificateOfService/document-download?selfPath={{ file.documentManagementURI }}&fileName={{ file.fileName }}">{{ file.fileName }}</a>
+                  </li>
+                {% endfor %}
+              </ul>
+            </div>
+          {% endif %}
+          {% if fileToUpload.value == 'PARTICULARS_OF_CLAIM' %}
```

Comment:
1. `===` even though it's not changing anything here. Just we shouldn't use `==`
2. `'PARTICULARS_OF_CLAIM'` is (or should be) some const. If it is - please use it here. Please see it in cmc-citizen-fe: 
- how to pass class to all njk views: https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/modules/nunjucks/index.ts#L92
- how to use it: https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/features/claim/views/interest-date.njk#L44

Reply:
update to use this style